docs/package_templates/cmake_package/all/test_package/CMakeLists.txt

```diff
@@ -1,12 +1,8 @@
 cmake_minimum_required(VERSION 3.15)
-
 project(test_package LANGUAGES C) # if the project is pure C
-# project(test_package LANGUAGES CXX) # if the project uses c++
+# project(test_package LANGUAGES CXX) # if the project uses C++
 
 find_package(package REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
-# don't link to ${CONAN_LIBS} or CONAN_PKG::package
 target_link_libraries(${PROJECT_NAME} PRIVATE package::package)
-# In case the target project need a specific C++ standard
-target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_17)
```

Comment:
I would keep these. The build is almost guaranteed to fail on older compiler versions if you don't specify the C++ standard for libraries targetting anything newer than C++11.

Reply:
Oh, wait, nevermind. Conan 2.x sets the C++ standard automatically.